assets/src/components/propertiesPanel/miniMap.tsx

```diff
@@ -0,0 +1,69 @@
+import { DomEvent } from "leaflet"
+import React, { useCallback, useContext } from "react"
+import { Link } from "react-router-dom"
+import { StateDispatchContext } from "../../contexts/stateDispatchContext"
+import { useStations } from "../../hooks/useStations"
+import { Vehicle, VehicleId } from "../../realtime"
+import { Shape, Stop } from "../../schedule"
+import { setSelectedVehicle } from "../../state/searchPageState"
+import inTestGroup, { MAP_BETA_GROUP_NAME } from "../../userInTestGroup"
+import { mapModeForUser } from "../../util/mapMode"
+import Map from "../map"
+
+const SearchMapLink = ({ vehicleId }: { vehicleId: VehicleId }) => {
+  const [, dispatch] = useContext(StateDispatchContext)
+
+  const openMapLinkRef = useCallback((node: HTMLAnchorElement | null) => {
+    if (node) {
+      DomEvent.disableClickPropagation(node)
+    }
+  }, [])
+
+  return (
+    <Link
+      ref={openMapLinkRef}
+      className="m-vehicle-properties-panel__map-open-link leaflet-bar"
+      to={mapModeForUser().path}
+      onClick={() => {
+        dispatch(setSelectedVehicle(vehicleId))
+      }}
+    >
+      Open Map
+    </Link>
+  )
+}
+
+const MiniMap = ({
+  vehicle,
+  shapes,
+  routeVehicles,
+}: {
+  vehicle: Vehicle
+  shapes: Shape[]
+  routeVehicles: Vehicle[]
+}) => {
+  const stations: Stop[] | null = useStations()
+
+  return inTestGroup(MAP_BETA_GROUP_NAME) ? (
+    <Map
+      selectedVehicleId={vehicle.id}
+      vehicles={[vehicle]}
+      shapes={shapes}
+      secondaryVehicles={routeVehicles}
```

Comment:
From the asana ticket:
> Any other vehicle currently on the map can be selected by clicking on it. This will display that vehicle’s vehicle properties card.

I don't think secondaryVehicles can be selected, is this critiera still relavant?

Reply:
My understanding is that criteria was referring to the main map page, not the mini map (and therefore not really in scope for this PR). I'll confirm that with design though before merging!

Reply:
Oh I misread this, I read `vehicle properties panel` somehow 😵‍💫